dvc/repo/ls.py

```diff
@@ -28,7 +28,7 @@ def ls(url, path=None, rev=None, recursive=None, dvc_only=False):
     """
     from . import Repo
 
-    with Repo.open(url, rev, subrepos=True, uninitialized=True) as repo:
+    with Repo.open(url, rev=rev, subrepos=True, uninitialized=True) as repo:
```

Comment:
aaand this is why we need more type checking :)

Reply:
Ok, type checking for `Repo.ls` -> `Repo.__init__` is not trivial enough to include into this PR, as it triggers a bunch of mypy reports that are unrelated to ls logic at all. Will need to get back to types later.